src/cli/test/reporters/github/index.ts

```diff
@@ -35,11 +40,29 @@ export default class GitHubReporter implements Pick<Reporter, 'onRunComplete'> {
           ? '`skuba test` passed.'
           : '`skuba test` found issues that require triage.';
 
+        if (coverage) {
+          const buildkiteOutput: string = [
+            '`skuba test` coverage:',
+            Buildkite.md.terminal(coverage),
+          ].join('\n\n');
+
+          await Buildkite.annotate(buildkiteOutput, {
```

Comment:
@samchungy how do you feel about having this reporter support both Buildkite and GitHub 😬

If we want coverage integrations across both, it may be wasteful to define two reporters and to run `renderCoverageText` twice.

Reply:
I don't love it but `renderCoverageText` is kinda costly I guess.

Do reporters run in series? I wonder if we could store it in memory? 🤔

If we do this -> might want to rename the reporter?